packages/frontend/src/MobileRoutes.tsx

```diff
@@ -204,15 +206,12 @@ const MobileRoutes: FC = () => {
                             </PrivateRoute>
                             <ProjectRoute path="/projects/:projectUuid">
                                 <Switch>
-                                    <Redirect
-                                        from="/projects/:projectUuid/saved/:savedQueryUuid/:mode?"
-                                        to="/minimal/projects/:projectUuid/saved/:savedQueryUuid"
-                                    />
-                                    <Redirect
-                                        from="/projects/:projectUuid/dashboards/:dashboardUuid/:mode?"
-                                        to="/minimal/projects/:projectUuid/dashboards/:dashboardUuid"
-                                    />
-
+                                    <Route path="/projects/:projectUuid/saved/:savedQueryUuid/:mode?">
+                                        <Navigate to="/minimal/projects/:projectUuid/saved/:savedQueryUuid" />
+                                    </Route>
+                                    <Route path="/projects/:projectUuid/dashboards/:dashboardUuid/:mode?">
+                                        <Navigate to="/minimal/projects/:projectUuid/dashboards/:dashboardUuid" />
+                                    </Route>
```

Comment:
the `Navigate` doesn't take the params as the `Redirect` did, this worked for me 

```suggestion
                               <Route
                                        path="/projects/:projectUuid/saved/:savedQueryUuid/:mode?"
                                        render={({ match }) => (
                                            <Navigate
                                                to={`/minimal/projects/${match.params.projectUuid}/saved/${match.params.savedQueryUuid}`}
                                                replace
                                            />
                                        )}
                                    />
                                    <Route
                                        path="/projects/:projectUuid/dashboards/:dashboardUuid/:mode?"
                                        render={({ match }) => (
                                            <Navigate
                                                to={`/minimal/projects/${match.params.projectUuid}/dashboards/${match.params.dashboardUuid}`}
                                                replace
                                            />
                                        )}
                                    />
```

Reply:
React router v6 doesn't provide the "match" or props in the render. Extracted to separate component with `useParams` and `CompatRoute` which is a compatible element between v5 and v6.